### DISCUSSION TITLE
vSphere IPI - Is it possible to overwrite the default haproxy configuration?

### DISCUSSION THREAD

**Discussion by danielchristianschroeter** [okd-project/okd#1348] [Q&A] [answered]
It it possible to overwrite the Ingress haproxy configuration from a vSphere IPI deployment?
I need to increase tune.bufsize and tune.maxrewrite

**Comment by jcpowermac** [comment, marked as answer]
https://docs.openshift.com/container-platform/4.8/release_notes/ocp-4-8-release-notes.html#ocp-4-8-configurable-headerBufferMaxRewriteByte-headerBufferBytes-parameters
